packages/terra-date-input/src/DateInput.jsx

```diff
@@ -677,7 +677,7 @@ class DateInput extends React.Component {
         {...this.props.dayAttributes}
         {...numberAttributes}
         refCallback={(inputRef) => { this.dayRef = inputRef; }}
-        label={label}
+        label={this.props.isInvalid ? `${'Error message'}, ${label}` : label}
```

Comment:
Same as above, 'Error message' should probably be internationalized.

Reply:
The string "Error message" is not required so it has been removed.